mors/lease_manager.py

```diff
@@ -67,9 +69,18 @@ def __init__(self, conf):
 
     def add_tenant_lease(self, context, tenant_obj):
         logger.info("Adding tenant lease %s", tenant_obj)
-        self.domain_mgr.add_tenant_lease(
+        if 'action' not in tenant_obj:
+            self.domain_mgr.add_tenant_lease(
             tenant_obj['tenant_uuid'],
             tenant_obj['expiry_mins'],
+            'power off',
```

Comment:
Can you just set action var in the if-else condition and keep a single call for add_tenant_lease? Also please add a constant called DEFAULT_ACTION = 'power off' instead of hardcoding it in these functions.

Reply:
done